ci/gpu/build.sh

```diff
@@ -29,22 +29,21 @@ export HOME=$WORKSPACE
 cd $WORKSPACE
 export GIT_DESCRIBE_TAG=`git describe --tags`
 export MINOR_VERSION=`echo $GIT_DESCRIBE_TAG | grep -o -E '([0-9]+\.[0-9]+)'`
-export FULL_VERSION=`echo $GIT_DESCRIBE_TAG | grep -o -E '([0-9]+\.[0-9]+\.[0-9]+)'`
-# Set `LIBCUDF_KERNEL_CACHE_PATH` environment variable to a unique path for
-# this CI job, e.g. `/tmp/cudf_0.12.0/gpu-v100-32-sm06-3-1234-jitify-cache`
-export LIBCUDF_KERNEL_CACHE_PATH="/tmp/cudf_$FULL_VERSION/\
-${NODE_NAME:+$NODE_NAME-}${EXECUTOR_NUMBER:+$EXECUTOR_NUMBER-}${BUILD_ID:+$BUILD_ID-}jitify-cache"
+# Set `LIBCUDF_KERNEL_CACHE_PATH` environment variable to $HOME/.jitify-cache because
+# it's local to the container's virtual file system, and not shared with other CI jobs
+# like `/tmp` is.
+export LIBCUDF_KERNEL_CACHE_PATH="$HOME/.jitify-cache"
 
 function remove_libcudf_kernel_cache_dir {
     EXITCODE=$?
-    echo "removing kernel cache dir: $LIBCUDF_KERNEL_CACHE_PATH"
-    rm -rf "$LIBCUDF_KERNEL_CACHE_PATH" || echo "could not rm -rf $LIBCUDF_KERNEL_CACHE_PATH"
+    logger "removing kernel cache dir: $LIBCUDF_KERNEL_CACHE_PATH"
+    rm -rf "$LIBCUDF_KERNEL_CACHE_PATH" || logger "could not rm -rf $LIBCUDF_KERNEL_CACHE_PATH"
```

Comment:
This isn't needed. $HOME (aka $WORKSPACE) is completely wiped at the start of each build.

Reply:
I would leave it though, what if they are running locally?

Reply:
Does your local scripts clean this directory when mounting and running in a container? If not then it's safe to leave it.

Reply:
If this isn't causing problems, I'd prefer that it stay in place a testing best practice.